NewsFeed/services/CCBRNewsDataStore.m

```diff
@@ -43,11 +44,20 @@ - (void)loadNextArticles {
         
         if (error) {
             // TODO: Handle error
+            self.error = error;
+            if (self.nextArticlesCallback) {
+                self.nextArticlesCallback(0, 0);
```

Comment:
It's better to create and use another callback to notify about the error. In this case, calling nextArticlesCallback with startIndex = 0 and endIndex = 0 might lead to incorrect logic in the callback handler code.